cli/CHANGELOG.md

```diff
@@ -17,12 +17,14 @@ _Released 07/15/2025 (PENDING)_
   - Removed support for Angular 17. The minimum supported version is now `18.0.0`. Addresses [#31303](https://github.com/cypress-io/cypress/issues/31303).
   - `@cypress/angular` now requires a minimum of `zone.js` `0.14.0`. Addresses [#31582](https://github.com/cypress-io/cypress/issues/31582).
   - The Cypress configuration wizard for Component Testing supports TypeScript 5.0 or greater. Addresses [#31187](https://github.com/cypress-io/cypress/issues/31187).
+  - `@cypress/vite-dev-server` is now an ESM only package. You will no longer be able to use this package from a CommonJS context. Addresses [#29557](https://github.com/cypress-io/cypress/issues/29557).
```

Comment:
Should this be here or in https://github.com/cypress-io/cypress/blob/develop/npm/vite-dev-server/CHANGELOG.md (or perhaps both?)

Reply:
@MikeMcC399 the `@cypress/vite-dev-server` should be auto generated via semantic commit. That being said, I need to mark this PR as a breaking change

Reply:
Thanks! Please mark this as resolved in that case!